components/form/test/form-element.js

```diff
@@ -0,0 +1,71 @@
+import '../../validation/validation-custom.js';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { FormElementMixin } from '../form-element-mixin.js';
+
+class FormElement extends FormElementMixin(LitElement) {
+
+	static get properties() {
+		return {
+			isValidationCustomValid: { type: Boolean },
+			tooltipMessage: { type: String },
+			value: { type: String, reflect: true }
+		};
+	}
+
+	constructor() {
+		super();
+		this.isValidationCustomValid = true;
+	}
+
+	render() {
+		return html`
+			<div>
+				<h1>Test Form Element</h1>
+				<input type="text" required/>
+				<d2l-validation-custom @d2l-validation-custom-validate=${this._validate} failure-text="Internal custom validation failed">
+				</d2l-validation-custom>
+			</div>
+		`;
+	}
+
+	updated(changedProperties) {
+		super.updated(changedProperties);
+
+		changedProperties.forEach((_, prop) => {
+			if (prop === 'value') {
```

Comment:
I can't entirely explain why but Lit's `.value=${this.myProp}` is doing something behind that scenes that doesn't trigger the underlying native elements validation. The property needs to be set via JavaScript to be recognized.

Reply:
Does this seem like a bug?  If so, maybe worth raising an issue in lit-html repo.  If you do, it would probable be good to link to a StackBlitz example (I have a Lit template if you need one).

Reply:
The issue here seemed to be caused by having `.value=${myProp}` called with an `undefined` `myProp` value. That seems to put the `input` in a bad state. Adding `ifDefined` fixed it.